packages/platform-sdk-profiles/src/profiles/profile.models.ts

```diff
@@ -64,6 +64,7 @@ export enum ProfileSetting {
 	UseMultiPeerBroadcast = "USE_MULTI_PEER_BROADCAST",
 
 	// UI
+	UseDevNetworks = "USE_DEVELOPMENT_NETWORKS",
```

Comment:
```suggestion
	UseTestNetworks = "USE_TEST_NETWORKS",
```

Networks internally are treated as live and test, not production and development. This is for consistency in naming with the majority outside of ARK coins.